lib/utils/asyncSerialMap.spec.js

```diff
@@ -0,0 +1,64 @@
+const { asyncSeriesMap } = require('./asyncSeriesMap')
+describe('asyncSerialMap', () => {
+  const nextTick = ms => new Promise((resolve) => {
```

Comment:
I'm not sure why **ms** is a parameter here if it's not used in the function.  Perhaps replace **ms** with **()**

Reply:
Ah, it used to be a wait function before I realised it could be `nextTick`.  I'll remove it.